Postman/Nfield Public API.postman_collection.json

```diff
@@ -10085,7 +10085,7 @@
 														"header": [
 															{
 																"key": "Content-Type",
-																"value": "application/octet-stream"
+																"value": "application/x-zip-compressed"
```

Comment:
This value belongs to the old endpoint. I think it is better if we not touch that. Just leave it octet-stream.